Makefile

```
@@ -14,12 +14,16 @@ help: ## This help.
 # thanks to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 .DEFAULT_GOAL := help
 
+build_mdi_svg: ## Build custom mdi svg
+	cd custom-npms/nc-mdi-svg && \
+	npm run build
```

```suggestion
	FONTAWESOME_PACKAGE_TOKEN=$(FONTAWESOME_PACKAGE_TOKEN) npm ci && \
	npm run build
```

please add:
* the install step
* the font awesome package token in order to be able to use this in build step in the future